integration/simulation/validate_chain.go

```diff
@@ -561,3 +565,17 @@ func assertNoDupeLogs(t *testing.T, logs []*types.Log) {
 		}
 	}
 }
+
+// Checks that the various APIs powering Obscuroscan are working correctly.
+func checkObscuroscan(t *testing.T, s *Simulation) {
+	for idx, client := range s.RPCHandles.RPCClients {
+		var totalTxs *big.Int
+		err := client.Call(&totalTxs, rpc.GetTotalTxs)
```

Comment:
Can we add this method to the ObscuroClient rather than using the raw RPC client? When we're calling from Go we should never have to work directly with the RPC calls imo.

Even if we're not sure how we want this method to look eventually, this RPC endpoint is publicly available so adding it to the client means we don't have to duplicate the calling logic and it's discoverable.